apps/prairielearn/src/schemas/schemas/infoAssessment.json

```
@@ -212,11 +216,12 @@
         },
         "role": {
           "description": "DEPRECATED -- do not use.",
-          "deprecated": true,
-          "enum": ["Student", "TA", "Instructor"]
+          "type": "string",
+          "enum": ["Student", "TA", "Instructor"],
+          "deprecated": true
         },
         "uids": {
-          "description": "A list of UIDs (like 'username@example.com', one of which is required for access",
+          "description": "A list of UIDs (like 'username@example.com'), one of which is required for access",
```

Let's change the corresponding Zod schema too:

```suggestion
          "description": "A list of UIDs (like 'username@example.com'), one of which is required for access.",
```

---

Addressed in 57735c1.